beacon-chain/sync/validate_beacon_attestation.go

```diff
@@ -352,31 +352,3 @@ func (s *Service) hasBlockAndState(ctx context.Context, blockRoot [32]byte) bool
 	hasState := hasStateSummary || s.cfg.beaconDB.HasState(ctx, blockRoot)
 	return hasState && s.cfg.chain.HasBlock(ctx, blockRoot)
 }
-
-func (s *Service) saveToPendingAttPool(att eth.Att) (pubsub.ValidationResult, error) {
```

Comment:
This function is no longer needed. Its only purpose was to create a dummy `SignedAggregateAttestationAndProof` and send it to the queue.